com.avocode.Avocode.yml

```diff
@@ -0,0 +1,60 @@
+id: com.avocode.Avocode
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "19.08"
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --system-talk-name=org.freedesktop.login1
```

Comment:
Are these two actually required? Application starts fine without them.

Reply:
Done. Yes, everything works fine without it. Thanks